app/Services/EmailListUpdater/EmailListUpdaterUsingDirectAdmin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\EmailListUpdater;
+
+use App\Models\Member;
+use App\Services\DirectAdmin\Contracts\EmailListAdapter;
+use App\Services\DirectAdmin\ValueObjects\EmailList;
+use App\Services\DirectAdmin\ValueObjects\EmailListSubscribers;
+
+final class EmailListUpdaterUsingDirectAdmin implements EmailListUpdater
+{
+    public function __construct(
+        private EmailList $list,
+        private EmailListAdapter $emailListAdapter,
+    ) {
+    }
+
+    public function update(): void
+    {
+        $emails = Member::query()
+            ->whereIn('soort', ['normaal', 'info'])
```

Comment:
Volgens mij moeten aspirant-leden hier ook bij... betekent alleen dat je eerste jaar geen contributie betaalt (en niet mag stemmen op de ALV), maar je wil natuurlijk wel dat ze alle mails ontvangen over bijv. kampen en uitjes enzo :)

Also, misschien overkill maar we zouden deze array in de config op kunnen nemen ipv hier (hoeft niet als env var, mag gewoon gecommit).